utils/abilities_events.cfg

```diff
@@ -5724,6 +5724,12 @@
                 name=did_succeed
                 greater_than=0
             [/variable]
+            [not]
+                [variable]
+                    name=bosses
+                    contains=$second_unit.id
+                [/variable]
+            [/not]
             [else]
```

Comment:
The code around is written quite stupidly, the condition says that if the result of a random number choice is greater than zero, then do nothing, otherwise start the redeem process. With your change, the unit also cannot be on the boss list for nothing to happen, so the result is that the unit is redeemed if it's not a boss or the random number was chosen correctly.

Reply:
Good point. Alright, here's a new commit where I unstupidified the code.